react/features/base/react/constants.js

```diff
@@ -5,3 +5,21 @@
  * everything, such as modal-type of components, or dialogs.
  */
 export const OVERLAY_Z_INDEX = 1000;
+
+/**
+ * The types of the buttons.
+ */
+export const BUTTON_TYPES = {
+    PRIMARY: 'primary',
+    SECONDARY: 'secondary',
+    DESTRUCTIVE: 'destructive'
```

Comment:
wasn't there a terciary one? Or re we net using it?

Reply:
Added it.